.github/workflows/build-game-bridge.yaml

```diff
@@ -0,0 +1,184 @@
+
+      # make artifacs avilable for download
+      - name: Upload Build Artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: game-bridge-artifacts
+          path: ./packages/game-bridge/dist
+
+      - name: Cache build artifacts
+        uses: actions/cache@v4
+        with:
+          path: ./packages/game-bridge/dist
+          key: "${{ runner.os }}-game-bridge-build-${{ env.TS_SDK_TAG }}"
+
+  create-unity-pr:
+    name: Create Unity PR
+    needs: build-game-bridge
+    if: ${{ github.event.inputs.game_engine == 'Unity' || github.event.inputs.game_engine == 'Both' }}
+    runs-on: ubuntu-latest-4-cores
+    steps:
+      - name: Checkout SDK Repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ env.GH_TOKEN }}
+          ref: "${{ env.TS_SDK_TAG }}"
+
+      - name: Setup Github
+        run: |
+          git config --global user.name "${GITHUB_ACTOR}"
+          git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+
+      - name: Restore Cached Build Artifacts
+        id: restore-cache-game-bridge-build
+        uses: actions/cache@v4
+        with:
+          path: ./packages/game-bridge/dist
+          key: ${{ runner.os }}-game-bridge-build-${{ env.TS_SDK_TAG }}
+
+      - name: Check Cache Restore
+        if: steps.restore-cache-game-bridge-build.outputs.cache-hit != 'true' 
+        run: echo "Game Bridge build cache not restored, exiting" && exit 1
+
+      - name: Checkout Unity SDK Repo
+        uses: actions/checkout@v4
+        with:
+          repository: immutable/unity-immutable-sdk
+          token: ${{ env.GH_TOKEN }}
+          path: unity-immutable-sdk
+          ref: main
+          clean: true
+          fetch-depth: 0
+
+      - name: Change Unity Branch
+        run: |
+          cd unity-immutable-sdk
+          git checkout -b "chore/update-game-bridge-${{ env.TS_SDK_TAG }}"
+          cd ..
+
+      - name: Copy Game Bridge Build
+        run: cp -r ./packages/game-bridge/dist/unity/index.html ./unity-immutable-sdk/src/Packages/Passport/Runtime/Resources/index.html
+
+      - name: Commit Changes
+        run: |
+          cd unity-immutable-sdk
+          git add src/Packages/Passport/Runtime/Resources/index.html
+          git commit -m "chore: update game bridge to ${{ env.TS_SDK_TAG }}"
+          cd ..
+
+      - name: and Push Changes
+        if: ${{ github.event.inputs.dry_run == false }}
+        run: |
+          cd unity-immutable-sdk
+          git push -u origin "chore/update-game-bridge-${{ env.TS_SDK_TAG }}"
+          cd ..
+
+      - name: Create PR (Dry Run)
+        if: ${{ github.event.inputs.dry_run == true }}
+        run: |
+          cd unity-immutable-sdk
+          echo "gh pr create --base main --head chore/update-game-bridge-${{ env.TS_SDK_TAG }} --title 'chore: update game bridge to ${{ env.TS_SDK_TAG }}' --body 'This PR updates the game bridge to ${{ env.TS_SDK_TAG }}'"
```

Comment:
Can the PR description include a link to the TS SDK tag/release version as well, please?